src/games/nogo_game.cc

```diff
@@ -5,7 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#pragma once
```

Comment:
Why remove this?

Reply:
This file is a `.cc` and it is never included. Having `#pragma once` makes a warning.